setup.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/python
```

Comment:
why to remove shebang? setup.py works very nicely as an executable script for many purposes.  And it must have a shebang if executable (as it is now)